src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -327,11 +329,15 @@ private fun HearingDto.toNomisUpdateHearing(): UpdateHearingRequest = UpdateHear
   internalLocationId = this.locationId,
 )
 
-private fun HearingOutcomeDto.toNomisCreateHearingResult(): CreateHearingResultRequest = CreateHearingResultRequest(
-  pleaFindingCode = this.plea!!.name,
-  findingCode = this.code.name,
-  adjudicatorUsername = this.adjudicator,
-)
+private fun HearingOutcomeDto.toNomisCreateHearingResult(adjudicationStatus: String): CreateHearingResultRequest =
+  CreateHearingResultRequest(
+    pleaFindingCode = this.plea!!.name,
+    findingCode = when (adjudicationStatus) {
+      "CHARGE_PROVED" -> "PROVED"
+      else -> adjudicationStatus
```

Comment:
I'll move this into a neater extension function if there is more mapping to do. Its more awkward as the source is no longer in the outcome dto too